lib/dice_bag/warning.rb

```diff
@@ -6,7 +6,9 @@ def initialize(template_filename)
     end
 
     def as_ruby_comment
-      lines.map { |line| "# #{line}".rstrip }.join("\n")
+      lines.map do |line|
+        prepend_ruby_comment_symbol(line) + "\n"
+      end.join
```

Comment:
IMHO having an extra method that in turn calls another extra method is a bit makes the code needlessly harder to follow, what about
```ruby
lines.map { |line| "#{line.dup.prepend("# ").rstrip}\n" }.join
```
(can remove `dup` if we don't care about mutating)

or even, keep what you had but add an extra `\n` to account for join not adding it?

```ruby
lines.map { |line| "# #{line}".rstrip }.join("\n") + "\n"
```

Reply:
Yeah thanks for pointing out, I was concerned about the cost of the redirection. Think the last one you suggested, append a `"\n"` in the end would make it clearer that we want to end with a new line, taking that approach 👍 (commit --amended)